packages/mdc-tabs/tab-bar/_mixins.scss

```diff
@@ -0,0 +1,8 @@
+@import "@material/theme/mixins";
+
+@mixin mdc-tab-bar-indicator-ink-color($color) {
+  .mdc-tab-bar__indicator,
+  &:not(.mdc-tab-bar-upgraded) .mdc-tab::after {
```

Comment:
I worry about & used within a comma separated selector.

Can you double check this `&:not(.mdc-tab-bar-upgraded) .mdc-tab::after` works with the css only version of tabs on our demo page?